website/content/api-docs/secret/identity/oidc-provider.mdx

```diff
@@ -240,4 +240,260 @@ $ curl \
+      "e": "AQAB"
+    }
+  ]}
+```
+
+## Authorization Endpoint
+
+Provides the [Authorization Endpoint](https://openid.net/specs/openid-connect-core-1_0.html#AuthorizationEndpoint)
+for an OIDC provider. This allows OIDC clients to request an authorization code
+to be used for the [Authorization Code Flow](https://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth).
+
+| Method      | Path                                      |
+| :---------- | :---------------------------------------- |
+| `GET/POST`  | `/identity/oidc/provider/:name/authorize` |
+
+### Parameters
+
+- `name` `(string: <required>)` - The name of the provider. This parameter is specified as part of the URL.
+
+- `scope` `(string: <required>)` - A space-delimited list of scopes to be requested. The `openid` scope is required.
+
+- `response_type` `(string: <required>)` - The OIDC authentication flow to be used. The following response types are supported: `code`.
+
+- `client_id` `(string: <required>)` - The ID of the requesting client.
+
+- `redirect_uri` `(string: <required>)` - The redirection URI to which the response will be sent.
+
+- `state` `(string: <required>)` - A value used to maintain state between the authentication request and client.
+
+- `nonce` `(string: <required>)` - A value that is returned in the ID token nonce claim. It is used to mitigate replay attacks.
+
+### Sample Request
+
+```shell-session
+$ curl \
+    --request GET \
+    --header "X-Vault-Token: ..." \
+    -G \
+    -d "response_type=code" \
+    -d "client_id=$CLIENT_ID" \
+    -d "state=af0ifjsldkj" \
+    -d "nonce=abcdefghijk" \
+    --data-urlencode "scope=openid" \
+    --data-urlencode "redirect_uri=http://127.0.0.1:8251/callback" \
+    http://127.0.0.1:8200/v1/identity/oidc/provider/test-provider/authorize
+```
+
+### Sample Response
+
+```json
+{
+  "code": "BDSc9kVYljxND93YpveBuJtSvguM3AWe",
+  "state": "af0ifjsldkj"
+}
+```
+
+## Token Endpoint
+
+Provides the [token endpoint](https://openid.net/specs/openid-connect-core-1_0.html#TokenEndpoint)
+for an OIDC provider.
+
+
+| Method  | Path                                  |
+| :------ | :------------------------------------ |
+| `POST`  | `/identity/oidc/provider/:name/token` |
+
+### Parameters
+
+- `name` `(string: <required>)` - The name of the provider. This parameter is
+specified as part of the URL.
+
+- `code` `(string: <required>)` - The authorization code received from the
+provider's authorization endpoint.
+
+- `grant_type` `(string: <required>)` - The authorization grant type. The
+following grant types are supported: `authorization_code`.
+
+- `redirect_uri` `(string: <required>)` - The callback location where the
+authorization request was sent. This must match the `redirect_uri` used when the
+original authorization code was generated.
+
+### Headers
```

Comment:
Nice! Like the headers sections.

Reply:
Thanks, I am thinking that they should be moved before the Parameters section.